src/main/webapp/app/core/login/login.service.ts

```diff
@@ -47,6 +49,30 @@ export class LoginService {
         });
     }
 
+    /**
+     * Login the user with SAML2.
+     * @param callback The callback function to use (optional)
+     */
+    loginSAML2(rememberMe: boolean, callback?: any) {
```

Comment:
can we avoid the type `any` here and directly add a proper type? Maybe something like a Promise?

Reply:
I ended up removing the callback since it isn't used

Reply:
@ivanchimeno so it's resolved?